src/QuantumToolbox.jl

```diff
@@ -1,5 +1,6 @@
 module QuantumToolbox
 
+using Pkg
```

Comment:
Maybe we can import `Pkg` instead of using. And maybe we can do it just in the related file rather then in the main file

Reply:
as we disussed in discord, I've updated it in d1b51e0